examples/advanced/create-react-app/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
```

Comment:
Should we make a favicon with the Style Dictionary logo?

Reply:
I wanted to keep the CRA app as similar as the original one as possible, so it's easier to see what has been updated for this example.